src/components/transport_manager/test/transport_adapter_test.cc

```diff
@@ -945,15 +952,23 @@ TEST_F(TransportAdapterTest, AbortedConnectSuccess) {
 TEST_F(TransportAdapterTest, SendData) {
   MockDeviceScanner* dev_mock = new MockDeviceScanner();
   MockServerConnectionFactory* serverMock = new MockServerConnectionFactory();
-  MockTransportAdapterImpl transport_adapter(
-      dev_mock, serverMock, NULL, last_state_, transport_manager_settings);
+  MockTransportAdapterImpl transport_adapter(dev_mock,
+                                             serverMock,
+                                             NULL,
+                                             last_state_wrapper_,
+                                             transport_manager_settings);
   SetDefaultExpectations(transport_adapter);
 
   EXPECT_CALL(*dev_mock, Init()).WillOnce(Return(TransportAdapter::OK));
   EXPECT_CALL(*serverMock, Init()).WillOnce(Return(TransportAdapter::OK));
   EXPECT_CALL(transport_adapter, Restore()).WillOnce(Return(true));
   transport_adapter.Init();
 
+  EXPECT_CALL(*dev_mock, Init()).WillOnce(Return(TransportAdapter::OK));
+  EXPECT_CALL(*serverMock, Init()).WillOnce(Return(TransportAdapter::OK));
+  EXPECT_CALL(transport_adapter, Restore()).WillOnce(Return(true));
+  transport_adapter.Init();
```

Comment:
Why is this init code here twice?

Reply:
@jacobkeeler Thank you. It is mistake of the last rebase. Removed in 5477ac1f48f21ba7a4312839e58907d64251d06a